clang/include/clang/Driver/Options.td

```diff
@@ -6543,9 +6543,10 @@ def fdepscan_include_tree : Flag<["-"], "fdepscan-include-tree">,
 //
 // FIXME: Add DepscanOption flag.
 def fdepscan_prefix_map_EQ : Joined<["-"], "fdepscan-prefix-map=">,
-    Group<f_Group>, MetaVarName<"<old>=<new>">,
+    Group<f_Group>, Flags<[CC1Option]>, MetaVarName<"<old>=<new>">,
```

Comment:
Nit: there's a comment above on line 6484 that says these options are "driver-only".  Could we move this one to a different section, or maybe just remove that comment if we don't care about the grouping anymore?

Reply:
Removed the "only" part in https://github.com/apple/llvm-project/pull/5965